src/actions/AuthActions.js

```diff
@@ -26,6 +27,17 @@ export function login(username, password) {
     return api
       .login(username, password)
       .then((response) => {
+        /**
+         * In local development and staging, we have the ability to manually
+         * set the formType by using URL params.
+         */
+        if (env.IsDevelopment() || env.IsStaging()) {
+          const params = location.search
+          const query = queryString.parse(params)
+          if (query.formType) {
+            window.formType = query.formType
```

Comment:
This is a little hacky because by the time the form data is retrieved in `ApplicationActions`, but URL params are gone. This is the workaround.

Reply:
seems fine enough for now!